benchmarks/index.md

```diff
@@ -1,60 +1,45 @@
+    url: /benchmarks/nyc_taxis
+  - name: PMC
+    url: /benchmarks/pmc
+  - name: Big5
+    url: /benchmarks/Big5
+  - name: Vectorsearch-nmslib-Cohere-1m
+    url: /benchmarks/vectorsearch-nmslib-cohere-1m
+  - name: Vectorsearch-nmslib-Cohere-10m
+    url: /benchmarks/vectorsearch-nmslib-cohere-10m
+  - name: Vectorsearch-faiss-Cohere-1m
+    url: /benchmarks/vectorsearch-faiss-cohere-1m
+  - name: Vectorsearch-faiss-Cohere-10m
+    url: /benchmarks/vectorsearch-faiss-cohere-10m
+  - name: Vectorsearch-lucene-Cohere-1m
+    url: /benchmarks/vectorsearch-lucene-cohere-1m
+  - name: Vectorsearch-lucene-Cohere-10m
+    url: /benchmarks/vectorsearch-lucene-cohere-10m
+  - name: Concurrent Segment Search Big5
+    url: /benchmarks/concurrent_segment_search_big5
+  - name: Concurrent Segment Search NYC_TAXIS
+    url: /benchmarks/concurrent_segment_search_nyc
+  - name: Concurrent Segment Search NOAA
+    url: /benchmarks/concurrent_segment_search_noaa
 benchmark_domain: 'benchmarks.opensearch.org'
-benchmark_dashboard_id: '88071cb0-f118-11ed-aff5-859eb6ed880f'
-benchmark_range_days: 7
-
-benchmark_height_desktop: 2000
-benchmark_height_mobile: 6000
 ---
-<style>
-    #benchmark-dashboard {
-        width: 100%;
-        height: {{ page.benchmark_height_desktop }}px;
-        background: transparent;
-        border: 0;
-    }
-
-    #navigation ul
-    {
-        margin: 0;
-        padding: 8px;
-    }
 
-    #navigation ul li
-    {
-        list-style-type: none;
-        display: inline-block;
-    }
-    
-    #navigation li:not(:first-child):before {
-        content: " | ";
-    }
-   
-    @media only screen and (max-width: 767px) {
-            #benchmark-dashboard {
-                height: {{ page.benchmark_height_mobile }}px;
-            }
-        }
-</style>
 <p>
-    Welcome to the OpenSearch performance benchmarking page. This page displays the results of ongoing performance testing for recent and upcoming versions of the OpenSearch software. You can view key performance metrics across different workloads with the dashboard visualizations below.
+    Welcome to the OpenSearch performance benchmarking page. You can see the results of ongoing performance testing for recent and upcoming versions of the OpenSearch software. You can view key performance metrics across different workloads with the dashboard visualizations by clicking on the dashboards link in the left column.
 </p>
+
 <h2>Why publish performance data?</h2>
 <p>
     The OpenSearch Project benchmarks the performance of OpenSearch releases to measure performance stability and gather data to inform software development. As an open-source project, we make this information publicly available in order to share it with the OpenSearch community.
 </p>
+
 <h2>Join the discussion</h2>
 <p>
     Questions or contributions? Connect with the OpenSearch community in the <a href="https://app.slack.com/client/T01QQ0Q5GMA/C0516H8EJ7R">#performance</a> channel on our public Slack.
```

Comment:
This line should probably link to https://opensearch.org/slack.html. This link didn't work for me.

Reply:
Updated.